recipes/physical_validation/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "physical_validation" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/shirtsgroup/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: dda33ba3a083fbb551192916b247effd12a954f4a1f03fa9b0aa3fd8d7c8016d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >3.6
+    - pip
+  run:
+    - python >3.6
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - matplotlib
```

Unless you need a PyQt-based GUI, better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base
```